web/client/epics/wfsquery.js

```diff
@@ -261,21 +263,11 @@ const viewportSelectedEpic = (action$, store) =>
             if (action.type === SELECT_VIEWPORT_SPATIAL_METHOD ||
                 action.type === CHANGE_MAP_VIEW && spatialFieldMethodSelector(store.getState()) === "Viewport") {
 
+                const projection = isLeaflet(store.getState()) ? 'EPSG:4326' : map.projection;
```

Comment:
bounds should also contain its srs, if it doesn't we should add it and not have knowledge of the mapping lib in use